protocol/0042-LIQF-setting_fees_and_rewarding_lps.md

```diff
@@ -109,13 +109,37 @@ The equity-like share for each LP is then
 (LP i equity-like share) = (LP i virtual stake) / (sum over j from 1 to N of (LP j virtual stake)).
 ```
 
-The average entry valuation (which should be reported by the APIs) is defined, at the time of change of an LP commitment as:
+There is a [Google sheet - requiring Vega login](https://docs.google.com/spreadsheets/d/14AgZwa6gXVBUFBUUOmB7Y9PsG8D4zmYN/edit#gid=886563806) showing this.
+
+The average entry valuation (which should be reported by the APIs and could be calculated only by the data node as it doesn't itself impact core state) is defined, at the time of change of an LP commitment as folows:
+1. There already is `average entry valuation` for the LP in question (and `average entry valuation = 0` for a new LP). The LP has existing physical stake `S` (and `S=0` for new LP) and wishes to add / remove stake `Delta S`. If `Delta S < 0` then `average entry valuation` is unchanged by the transaction. If `S + Delta S = 0` then the LP is exiting their LP commitment and we do not calculate the `average entry valuation` for them in this case. 
+So `Delta S > 0` (and so `S+Delta S > 0`) in what follows.
+2. Calculate the entry valuation at the time stake `Delta S` is added / removed as 
+```
+(entry valuation) = sum over j from 1 to N of (LP j virtual stake)
+```
+3. Update the average entry valuation to 
+```
+(average entry valuation) <- (average entry valuation) x S / (S + Delta S) + (entry valuation) x (Delta S) / (S + Delta S)
+```
+Example 1: 
+1. A new LP has `0` stake and they wish to add `Delta S = 100`. Currently the sum of all virtual stakes is `1000`. The average entry valuation is
 ```
-(Avg entry valuation i) = sum over j from 1 to N of (LP j virtual stake)
+(average entry valuation) <- 0 + 1000 x 100 / (0 + 100) = 1000
+```
+Example 2:
+An existing LP has `average entry valuation 1000` and `S=100`. Currently the sum of all virtual stakes is `2000`. They wish to add `10` to their stake. 
+```
+(average entry valuation) <- 1000 x 100 / (100 + 10) + 2000 x 10 / (100 + 10) = 1090.9.... 
+```
+Example 3: 
+An existing LP has `average entry valuation 1090.9` and `S=110`. Currently the sum of all virtual stakes is `3000`. They wish to remove `20` from their stake. Their average entry valuation stays the same
+```
+(average entry valuation) = 1090.9
 ```
 
 
-There is a [Google sheet - requiring Vega login](https://docs.google.com/spreadsheets/d/14AgZwa6gXVBUFBUUOmB7Y9PsG8D4zmYN/edit#gid=886563806) showing this.
+
 
 
 **Check** the sum from over `i` from `1` to `N` of `LP i equity-like share` is equal to `1`.
```

Comment:
Move this "Check" line above section defining average-entry-valuation formulas so it is straight after ELS formulas.